canopen_monitor/parser/sync.py

```diff
@@ -2,6 +2,6 @@
 
 
 def parse(data: bytes):
-    if len(data) > 1:
-        raise ValueError("SYNC message is outside of bounds")
+    # if len(data) > 1:
```

Comment:
We should replace this with an inline notification in the app instead of throwing an error.